addons/hr_expense/tests/test_expenses.py

```diff
@@ -1,907 +1,245 @@
+            {'balance':  1391.30, 'account_id': self.expense_account.id,    'name': 'expense_employee: PA 2*800 + 15%',   'date': date(2021, 10, 10)},
+            {'balance':    18.46, 'account_id': tax_account_id,             'name': '15%',                                'date': date(2021, 10, 10)},
+            {'balance':    18.46, 'account_id': tax_account_id,             'name': '15% (Copy)',                         'date': date(2021, 10, 10)},
+            {'balance':   208.70, 'account_id': tax_account_id,             'name': '15%',                                'date': date(2021, 10, 10)},
+            {'balance': -1760.00, 'account_id': default_account_payable_id, 'name': False,                                'date': date(2021, 10, 10)},
+
+            # company_account expense 1 move
+            {'balance':   869.57, 'account_id': product_c_account_id,       'name': 'expense_employee: PC 1000 + 15%',    'date': date(2021, 10, 12)},
+            {'balance':   130.43, 'account_id': tax_account_id,             'name': '15%',                                'date': date(2021, 10, 12)},
+            {'balance': -1000.00, 'account_id': company_payment_account_id, 'name': 'expense_employee: PC 1000 + 15%',    'date': date(2021, 10, 12)},
+
+            # company_account expense 2 move
+            {'balance':  123.08, 'account_id': product_b_account_id,        'name': 'expense_employee: PB 160 + 2*15% 2', 'date': date(2021, 10, 12)},
+            {'balance':   18.46, 'account_id': tax_account_id,              'name': '15%',                                'date': date(2021, 10, 12)},
+            {'balance':   18.46, 'account_id': tax_account_id,              'name': '15% (Copy)',                         'date': date(2021, 10, 12)},
+            {'balance': -160.00, 'account_id': company_payment_account_id,  'name': 'expense_employee: PB 160 + 2*15% 2', 'date': date(2021, 10, 12)},
+        ])
 
-        for _i in range(2):
-            self.env['hr.expense'].create({
-                'name': 'Car Travel Expenses',
-                'employee_id': self.expense_employee.id,
-                'product_id': self.product_c.id,
-                'total_amount': 350.00,
-                'tax_ids': [(6, 0, [self.tax_purchase_a.id])],
-                'sheet_id': expense.id,
-                'analytic_distribution': {str(self.analytic_account_1.id): 100},
+        # Own_account partial payment
+        payment_1 = self.get_new_payment(expense_sheet_by_employee, 1700.0)
+        liquidity_lines1 = payment_1._seek_for_lines()[0]
+        self.assertRecordValues(expense_sheet_by_employee, [{'payment_state': 'partial', 'state': 'done'}])
+
+        # Test printing checks if the module is installed
+        if self.env['ir.module.module'].search([('name', '=', 'account_check_printing'), ('state', '=', 'installed')], limit=1):
```

Comment:
should be tested in bridge module, and if there isn't any bridge module, only values of payments (that would be printed) should be checked.

the whole "if module is installed" should be removed